docs/code/transports/ServiceBusManagedIdentityConsoleListener.cs

```diff
@@ -20,8 +20,7 @@ await Host.CreateDefaultBuilder(args)
                     {
                         var settings = new HostSettings
                         {
-                            ServiceUri = new Uri("sb://your-service-bus-namespace.servicebus.windows.net"),
-                            TokenCredential = new DefaultAzureCredential() // From Azure.Identity.dll
+                            ServiceUri = new Uri("sb://your-service-bus-namespace.servicebus.windows.net")
```

Comment:
So, this is kind of what I was trying to say in the chat. There should be **NO** reason to use HostSettings, and it needs to be stripped from the documentation entirely. 

It's just:

```cs
cfg.Host(uri);
```

OR

```cs
cfg.Host(uri, h => h.Something = something);
```

Reply:
Ah - that was just a daft change on my part. I didn't actually use this way! will sort.